src/node/desktop/package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "clean": "ts-node scripts/clean.ts",
     "lint": "eslint ./src ./test",
-    "make": "wrap () { set -e ; yarn ; yarn run clean ; electron-forge make $*; ./scripts/rename-dmg.sh;}; wrap",
+    "make": "wrap () { set -e; yarn; yarn run clean; electron-forge make $*; }; wrap",
```

Comment:
See other comment about potentially just using `electron-forge package` and skipping `make` altogether.